pep-0683.rst

```diff
@@ -0,0 +1,342 @@
+PEP: 683
+Title: Immortal Objects, Using a Fixed Refcount
+Author: Eric Snow <ericsnowcurrently@gmail.com>, Eddie Elizondo <eelizondo@fb.com>
+PEP-Delegate: <TBD>
+Discussions-To: python-dev@python.org
+Status: Draft
+Type: Informational
+Created: 10-02-2022
+Python-Version: 3.11
+Post-History:
+Resolution: <TBD>
+
+
+Abstract
+========
+
+Under this proposal, any object may be marked as immortal.
+"Immortal" means the object will never be cleaned up (at least until
+runtime finalization).  Specifically, the refcount for an immortal
```

Comment:
Minor: You never link / explain what the refcount is, although it may be assumed knowledge for the type of people reading this PEP.